apps/demo/emails/raycast-magic-link.tsx

```diff
@@ -0,0 +1,126 @@
+import { Img } from '@react-email/img';
+import { Link } from '@react-email/link';
+import { Preview } from '@react-email/preview';
+import { Section } from '@react-email/section';
+import { Text } from '@react-email/text';
+import * as React from 'react';
+
+export default function Email() {
+  return (
+    <Html>
+      <Head />
+      <Preview>
+        Log in with this magic link.
+      </Preview>
+      <Section style={main}>
+        <Container style={container}>
+          <Img
+            src="https://miro.medium.com/max/1024/1*ThvbH5hKHEM4jpuAqRNfBQ.png"
+            width="170"
+            height="50"
+            alt="Koala"
+            style={logo}
+          />
+          <Container style={container2}>
+            <Text style={heading}>🪄 Your magic link</Text>
+            <Link
+            href="https://www.raycast.com"
+            target="_blank"
+            style={{
+              fontSize: '22px',
+              display: 'block',
+              marginBottom: '16px',
+              marginTop: '50px',
+            }}
+            >
+                👉 Click here to sign in 👈
+            </Link>
+            <Text style={paragraph}>
+                If you didn't request this, please ignore this email.
+            </Text>
+            <Text style={paragraph2}>
+                Best,
+                <br />
+                - Raycast Team
+            </Text>
+            <Hr style={hr} />
+            <Img
+                src="https://miro.medium.com/max/1024/1*ThvbH5hKHEM4jpuAqRNfBQ.png"
+                width={50}
+                style={{
+                    WebkitFilter: 'grayscale(100%)',
+                    filter: 'grayscale(100%)',
+                    marginBottom: '35px'
+                }}
+            />
+            <Text style={footer}>Raycast Technologies Inc.</Text>
+            <Text style={footer}>2093 Philadelphia Pike #3222, Claymont, DE 19703</Text>
+          </Container>
+        </Container>
+      </Section>
+    </Html>
+  );
+}
+
+const fontFamily =
+  '-apple-system,BlinkMacSystemFont,"Segoe UI",Roboto,Oxygen-Sans,Ubuntu,Cantarell,"Helvetica Neue",sans-serif';
+
+const main = {
+    backgroundColor: '#ffffff',
+    backgroundImage: 'url(https://ci5.googleusercontent.com/proxy/aK4MHkXhOLhD-fBGFTqQ1eYuKrX9vz7X4KqU86__z0ZVgPhYRO1eShT3hsiX5jH82wBpnAt6hEb1uHoeegzT3BHoeENi3YoxNVpkdOivKOC_KRhVHM2ezc-Gv1jELM-pzLVWh-P5QUp8kUZ3JfdhPICf21gM66NgoIbJW-_i6lWnKkRqvpI=s0-d-e1-ft#http://cdn.mcauto-images-production.sendgrid.net/ef379b8a298c6d3d/deb93c11-8a82-47c2-bcf5-4cdee8d820a6/1440x724.png), url("https://ci4.googleusercontent.com/proxy/gvEo8QunL-goa6twd_HDizIGrITPDyNeio0oZ5NO9cZDBt7wWGiz0mz3hD_XzF6tizUwweII5iPYrP-c7DqgJhKVwD6jhJmgHxRapKaiPNq4x80zLBK1OTZqsiMap4eKCiQ4Tix_1EnEJW2znxKDASZZWKP51uyU00plVeabi7hFafUIhaU=s0-d-e1-ft#http://cdn.mcauto-images-production.sendgrid.net/ef379b8a298c6d3d/51f59eca-c148-46b7-8037-58de51b9a0da/1440x440.png")',
+    backgroundPosition: 'bottom',
```

Comment:
Background image should be located in the Container:

<img width="1351" alt="CleanShot 2022-12-21 at 07 06 40@2x" src="https://user-images.githubusercontent.com/398893/208878907-5b7725af-d649-4ffd-ab84-3b396d6cbf8d.png">

Reply:
Hey, you have uploaded only the background image for the top right corner.

Please upload [this](https://ci4.googleusercontent.com/proxy/gvEo8QunL-goa6twd_HDizIGrITPDyNeio0oZ5NO9cZDBt7wWGiz0mz3hD_XzF6tizUwweII5iPYrP-c7DqgJhKVwD6jhJmgHxRapKaiPNq4x80zLBK1OTZqsiMap4eKCiQ4Tix_1EnEJW2znxKDASZZWKP51uyU00plVeabi7hFafUIhaU=s0-d-e1-ft#http://cdn.mcauto-images-production.sendgrid.net/ef379b8a298c6d3d/51f59eca-c148-46b7-8037-58de51b9a0da/1440x440.png) image for the bottom left corner too so that i can fix this issue